src/test/java/com/battleship/backend/TestClasses.java

```diff
@@ -80,6 +80,19 @@ public void addSection(Sectionable section, int row, int col) {
 
             }
 
+        public boolean shipListEquals(ArrayList<Ship> otherShipList){
+            int count = 0;
+            for(Ship boardShip : shipList){
+                for(Ship otherShip : otherShipList){
+                    if(boardShip.equals(otherShip)){
+                        count += 1;
+                    }
+
+                }
+            }
+            return count == shipList.size();
+        }
```

Comment:
This method looks very much like the actual method.  Can we simplify for the test?  If the methods are the same why have a test class?